astropy/table/tests/test_table.py

```diff
@@ -1873,6 +1873,16 @@ def test_masking(self):
                 else:
                     assert column.byteswap().newbyteorder().dtype == t2[name].dtype
 
+    def test_units(self):
```

Comment:
You need to skip the test if there is no pandas available.
```
@pytest.mark.skipif('not HAS_PANDAS')
```

Reply:
I thought the whole class is skipped if `not HAS_PANDAS`?

This is a method in

```
@pytest.mark.skipif('not HAS_PANDAS')
class TestPandas:
```

Reply:
Ah right, never mind.